### PR TITLE
Added config for choosing host selection policy

### PR DESCRIPTION
**Pull request by dahankzter**
This PR allows for choosing the host selection policy on the command line.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -370,3 +382,14 @@ func main() {
 			"\n  mean:\t\t", time.Duration(result.Latency.Mean()))
 	}
 }
+
+func getHostSelectionPolicy(hostSelectionPolicy string, hosts []string) gocql.HostSelectionPolicy {
```
Comment (mmatczuk) [inline main.go]: How about `func newHostSelectionPolicy(name string, hosts []string)) gocql.HostSelectionPolicy`?

File: main.go [hunk 2]
```diff
@@ -370,3 +382,14 @@ func main() {
...
+	switch hostSelectionPolicy {
+	case "round-robin":
+		return gocql.RoundRobinHostPolicy()
+	case "host-pool":
+		return gocql.HostPoolHostPolicy(hostpool.New(hosts))
+	default:
```
Comment (mmatczuk) [inline main.go]: Default should report an error, missing case for token aware.

Comment (dahankzter) [reply main.go]: sure

File: main.go (hunk 1 above)
Comment (dahankzter) [reply main.go]: Looks good.

File: main.go [hunk 3]
```diff
@@ -370,3 +387,16 @@ func main() {
 			"\n  mean:\t\t", time.Duration(result.Latency.Mean()))
 	}
 }
+
+func newHostSelectionPolicy(hostSelectionPolicy string, hosts []string) (gocql.HostSelectionPolicy, error) {
```
Comment (mmatczuk) [inline main.go]: name?

**Pull request merged**